src/ui-kit/components/pagination/pagination.template.html

```diff
@@ -3,16 +3,16 @@
     <li *ngIf="showPrevious()"><a href="javascript:void(0)" class="page-previous" aria-label="previous page"
             (click)="onPreviousClick()" (keyup.enter)="onPreviousClick()">&lsaquo; Prev </a></li>
     <li><a href="javascript:void(0)" class="page-button" [ngClass]="textDecoration(1)" (click)="onPageClick(1)"
-            [attr.aria-label]="getAriaLabel(1)" (keyup.enter)="onPageClick(1)"><span class="sr-only">Page</span>1</a>
+            [attr.aria-label]="getAriaLabel(1)" (keyup.enter)="onPageClick(1)">1</a>
     </li>
     <li *ngIf="showFirstEllipsis()"><span class="first-ellipsis">&hellip;</span></li>
     <li *ngFor="let i of consecutivePageRange()"><a href="javascript:void(0)" class="page-button"
             [ngClass]="textDecoration(i)" (click)="onPageClick(i)" [attr.aria-label]="getAriaLabel(i)"
-            (keyup.enter)="onPageClick(i)"><span class="sr-only">Page</span>{{i}}</a></li>
+            (keyup.enter)="onPageClick(i)">{{i}}</a></li>
```

Comment:
This was the cause for the issue as prev and next links were working just fine. This likely interfered with how JAWS processes href link. This span tag was placed for screen readers to prefix the page number with the word `Page`. This fix moves the text to the aria-label attribute by modifying getAriaLabel method.